projects/Allwinner/options

```diff
@@ -62,3 +62,6 @@
   # debug tty path
     DEBUG_TTY="/dev/console"
 
+  # set the addon project
+    ADDON_PROJECT="H3"
+
```

Comment:
small nit: probably no need for the extra new line, none of the other project options files updated has a blank line